docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+# Configuration file to run ci tests in docker containers
+#
+# Defines a database container with MySQL and a container with the environment
+# to build libdrizzle-redux. The latter container runs either ubuntu or centos
+# depending on the travis build configuration.
+#
+# See https://docs.docker.com/compose/overview/ for more information about
+# the format of the docker compose config file
+version: '3'
+
+services:
+  db:
+    image: mysql:5.7
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: pass
+      MYSQL_DATABASE: test
+      MYSQL_USER: user
+      MYSQL_PASSWORD: pass
```

Comment:
Why this doesn't use the `.env` file? DRY.

Reply:
It should. Some env variables are not relevant for the db container. Multiple files can be specified for `env_file` so will separate the vars into shared and container specific.